README.md

```diff
@@ -1,3 +1,7 @@
 # LabaHomework2
 
-Real Estate Agency structure(classes only)
+Real Estate Agency structure
```

Comment:
Can you plz describe more details on what is this project for, which classes your reviewers should pay special attention, what can user do with your app.